### PR TITLE
[docs] Move pages for C, C++, Ruby , PHP and Scala to Drivers and ORMs

### LINKED ISSUE #14273: [docs] Move Build an app pages for C, C++, Ruby, PHP, Scala to Drivers and ORMs
### Description

Move the files to comply with the format from PR: https://github.com/yugabyte/yugabyte-db/pull/14214

### PR DESCRIPTION
**Pull request by aishwarya24**
resolves #14273 
Summary
- Moved C, C++, Ruby , PHP and Scala to Drivers and ORMs from Build an app section
- Added their entries to the specific landing pages .
- redirects in netlify.toml( need to check if this can be simplified)

Next steps
1. Move all the **YSQL** pages for Java, Go, Python, C#, Rust, and Node.js to Drivers and ORMs.
2. Move/Remove duplicates from integrations
3. Standardize the content on all the YSQL and YCQL pages under Drivers and ORMs by updating the existing content for the newly moved pages.
4. The table entries on the main landing page as well as under individual languages need updates.

### CODE REVIEW

File: docs/content/preview/drivers-orms/_index.md [hunk 1]
```diff
@@ -92,4 +105,23 @@ The following libraries are officially supported by YugabyteDB.
...
 | :--------- | :------------ | :----------- |
-| [Diesel](rust/diesel/) | Full | [Diesel](rust/diesel/) |
+| [Diesel](rust/diesel/) | Full | [CRUD example](rust/diesel/) |
+
+### Ruby
+
+| Driver/ORM | Support Level | Example apps |
+| :--------- | :------------ | :----------- |
+| [Pg Gem Driver](ruby/ysql-pg/) | Full | [CRUD example](ruby/ysql-pg/) |
+| [YugabyteDB Ruby Driver for YCQL](ruby/ycql/) | Full | [CRUD example](ruby/ycql/) |
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/_index.md:115]: missing some kind of ORM?

File: docs/content/preview/drivers-orms/c/_index.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+- **Machine and Software requirements**
+  - a 32-bit (x86) or 64-bit (x64) architecture machine.
+  - gcc 4.1.2 or later, clang 3.4 or later installed.
+
+- **YugabyteDB cluster**
+  - Create a free cluster on [YugabyteDB Managed](https://www.yugabyte.com/cloud/). Refer to [Use a cloud cluster](../../quick-start-yugabytedb-managed/). Note that YugabyteDB Managed requires SSL.
+  - Alternatively, set up a standalone YugabyteDB cluster by following the steps in [Install YugabyteDB](../../quick-start/).
+
+## Next steps
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/c/_index.md:37]: think we gave these Next Steps the chop on other pages, pretty redundant

File: docs/content/preview/drivers-orms/php/_index.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+## Prerequisites
+
+To develop C applications for YugabyteDB, you need:
+
+- **a YugabyteDB cluster**
+  - Create a free cluster on [YugabyteDB Managed](https://www.yugabyte.com/cloud/). Refer to [Use a cloud cluster](../../quick-start-yugabytedb-managed/). Note that YugabyteDB Managed requires SSL.
+  - Alternatively, set up a standalone YugabyteDB cluster by following the steps in [Install YugabyteDB](../../quick-start/).
+
+## Next steps
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/php/_index.md:33]: Cut

File: docs/content/preview/drivers-orms/rust/_index.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ menu:
   preview:
     identifier: rust-drivers
     parent: drivers-orms
-    weight: 570
+    weight: 600
 type: indexpage
 ---
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/rust/_index.md:14]: Cut Next Steps

File: docs/content/preview/drivers-orms/scala/_index.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+## Prerequisites
+
+To develop Scala applications for YugabyteDB, you need:
+
+- **a YugabyteDB cluster**
+  - Create a free cluster on [YugabyteDB Managed](https://www.yugabyte.com/cloud/). Refer to [Use a cloud cluster](../../quick-start-yugabytedb-managed/). Note that YugabyteDB Managed requires SSL.
+  - Alternatively, set up a standalone YugabyteDB cluster by following the steps in [Install YugabyteDB](../../quick-start/).
+
+## Next steps
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/scala/_index.md:33]: Cut

**Review by ddhodge — APPROVED** [review]
Misc comments

File: docs/content/preview/drivers-orms/_index.md (hunk 1 above)
Comment (aishwarya24) [reply docs/content/preview/drivers-orms/_index.md:115]: Yes ActiveRecord ORM. Thanks .

**Pull request merged**